tests/test_opening.py

```diff
@@ -1205,6 +1205,41 @@ def test_funder_contribution_limits(node_factory, bitcoind):
     assert l3.daemon.is_in_log(r'calling `signpsbt` .* 7 inputs')
 
 
+@pytest.mark.openchannel('v2')
+@pytest.mark.developer("requres 'dev-disconnect'")
+def test_inflight_dbload(node_factory, bitcoind):
+    """Bad db field access breaks Postgresql on startup with opening leases"""
```

Comment:
extremely minor nit: technically it only breaks if you try to load inflights from a postgres db, which happens in the case that a channel open isn't resolved at startup